src/Components/ItemList/ItemList.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Item from '../Item/Item';
+
+
+const ItemList = ({todoItem}) => (
+	<ul>
+    	<li> <Item todoItem={todoItem} />  </li>
+    	<li> <Item todoItem={'To Write props'} /> </li>
+    	<li> <Item todoItem={'To finish all tasks'} /> </li>
```

Comment:
Codacy found an issue: [Mixed spaces and tabs.](https://app.codacy.com/gh/WebHeroSchool/todo_react_project/pullRequest?prid=5893382)